server/src/main/java/com/defold/extender/AsyncBuilder.java

```diff
@@ -63,25 +65,25 @@ private void writeExtenderLogsToFile(Extender extender, File file) {
         }
     }
 
-    private void writeExceptionToFile(Exception e, File file) {
+    private void writeExceptionToFile(Exception exception, File file) {
         try {
             PrintWriter writer = new PrintWriter(file);
-            e.printStackTrace(writer);
+            exception.printStackTrace(writer);
             writer.close();
         }
         catch(Exception e) {
             LOGGER.error("Could not write exception to error file", e);
         }
     }
-    
+
     @Async
-    public void asyncBuildEngine(MetricsWriter metricsWriter, String platform, String sdkVersion, 
+    public void asyncBuildEngine(MetricsWriter metricsWriter, String platform, String sdkVersion,
             File jobDirectory, File uploadDirectory, File buildDirectory) throws IOException {
         String jobName = jobDirectory.getName();
         Thread.currentThread().setName(String.format("async-build-%s", jobName));
         File resultDir = new File(jobResultLocation.getAbsolutePath(), jobName);
         resultDir.mkdir();
-        Extender extender;
+        Extender extender = null;
```

Comment:
Isn't it null by default in Java?

Reply:
Well, no, it is considered "uninitialized" which means that I get an error when trying to pass it as an argument to the helper function.